test/test.spec.js

```diff
@@ -243,4 +243,28 @@ describe('angular preset', function() {
         done();
       }));
   });
+
+  it('should support directly utilizing http/https repository urls', function(done) {
+    preparing(8);
+    let i = 0;
+
+    conventionalChangelogCore({
+      config: preset,
+      pkg: {
+        path: __dirname + '/fixtures/bitbucket-http-host.json',
+      },
+    }).on('error', done).pipe(through(function(chunk, enc, cb) {
+      chunk = chunk.toString();
+
+      expect(chunk).to.include('/compare/');
```

Comment:
From this test, it's not obvious that the entire repo URL is correctly being used in the output. Can you adjust it be something like:
```
expect(chunk).to.include('http://any.bbucket.host:7999/projects/proj/repos/repo-name/compare/)
```
... where the URL will be the one from the fixture? That makes it clearer that the system is using the entire repo URL from package.json.